index.js

```diff
@@ -0,0 +1,118 @@
+// DOM ELEMENTS
+const likeButton = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+const form = document.querySelector("#new-goal-form")
+const doc = document.querySelector("body")
+doc.style.background = "AliceBlue"
+// console.log(form)
+
+// EVENT HANDLERS
+// made helper method for likes event listener. have to put fetch inside this method so it recognizes my newLikes variable. I'm learning!!
+const addLikes = () => {
+  let newLikes = parseInt(likes.textContent) + 1
+  likes.textContent = `${newLikes} Likes`
+
+  fetch("http://localhost:3000/players/1", {
+    method: 'PATCH',
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    },
+    body: JSON.stringify({
+      likes: newLikes
+    })
+  })
+    .then(response => response.json())
+    .then(updatedPlayer => renderPlayer(updatedPlayer))
+  // i would prefer to update my DOM from within the .then but I'm not confident in it. I tend to break things in new and creative ways!
+  // update: figuring stuff out, this is awesome!!!
+}
+
+
+// similarly, made helper method for form event listener. have to put fetch inside this method so it recognizes my newGoal variable.
+const submitGoal = (event) => {
+  event.preventDefault()
+
+  const newGoal = {
+    playerId: 1,
+    link: event.target["goal-link"].value,
+    description: event.target["goal-description"].value
```

Comment:
You can also use the `name=` attribute of the inputs to access them from the form:

```js
        link: event.target.link.value,
        description: event.target.description.value
```